js/utils.js

```diff
@@ -1184,6 +1184,10 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
     var timeRegex = /^[0-9]{1,2}:[0-9]{1,2}$/
     
     if (valueType === 'string') {
+      if (!isNaN(value)) {
```

Comment:
![image](https://user-images.githubusercontent.com/290733/102083657-d25ffb00-3e0b-11eb-9efe-5a39c0a3f89c.png)

`null` is not a number.

Reply:
Yes , but this condition works only when `typeof value === 'string'`